scene/3d/mesh_instance.cpp

```diff
@@ -96,7 +96,7 @@ void MeshInstance::_get_property_list(List<PropertyInfo> *p_list) const {
 	ls.sort();
 
 	for (List<String>::Element *E = ls.front(); E; E = E->next()) {
-		p_list->push_back(PropertyInfo(Variant::REAL, E->get(), PROPERTY_HINT_RANGE, "0,1,0.01"));
+		p_list->push_back(PropertyInfo(Variant::REAL, E->get(), PROPERTY_HINT_RANGE, "0,1,0.001,or_lesser"));
```

Comment:
I know that's what @reduz suggested, but AFAIK this won't work as expected. `or_lesser` means that we can now go below 0, but the step stays 0.001. So we can now have -5.002, but not 0.0003.

Reply:
#19242 needs to be addressed to have the possibility to have higher precision without changing the default step.

Reply:
I'll change it back to my patched value of:

```cpp
p_list->push_back(PropertyInfo(Variant::REAL, E->get(), PROPERTY_HINT_RANGE, "0,1,0.00001"));
```